Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx

```diff
@@ -46,49 +52,81 @@ export const ProfilePageJob = () => {
-        else
+        if(isCandidate){
+            // setExperience(experience.slice(0, currentExperience).concat(experience.slice(currentExperience + 1, experience.length)));
+            delete_experience(bufferId);
+        }       
+        else{
             setJobPostings(jobPostings.slice(0, currentExperience).concat(jobPostings.slice(currentExperience + 1, jobPostings.length)));
+        }
+        window.location.reload();
+
     };
 
     const handleSaveExperienceDialog = () => {
         setOpenDialog(false);
-        let e = jobPostings;
+        // let e = jobPostings;
 
+        // if(isCandidate){
+        //     e = experience;
+        // }
+        // e[currentExperience]["title"] = bufferTitle;
+        // e[currentExperience]["company"] = bufferCompany;
+        // e[currentExperience]["start_date"] = bufferStartDate;
+        // e[currentExperience]["end_date"] = bufferEndDate;
+        // e[currentExperience]["description"] = bufferDescription;
         if(isCandidate){
-            e = experience;
+            // setExperience(e)
+            console.log(experience)
+            update_experience(bufferId,{
+                "title": bufferTitle,
+                "company": bufferCompany,
+                "start_date": bufferStartDate,
+                "end_date": bufferEndDate,
+                "description": bufferDescription,
+            }
+        );
+
+        }
+        else{
+            // setJobPostings(e);
         }
-        e[currentExperience]["title"] = bufferTitle;
-        e[currentExperience]["company"] = bufferCompany;
-        e[currentExperience]["start_date"] = bufferStartDate;
-        e[currentExperience]["end_date"] = bufferEndDate;
-        e[currentExperience]["description"] = bufferDescription;
-        if(isCandidate)
-            setExperience(e)
-        else
-            setJobPostings(e);
+            
+        window.location.reload();
     };
 
     const handleAddExperienceDialog = () => {
         setOpenAddDialog(false);
         if(isCandidate){
-            setExperience([...experience, {
+            // setExperience([...experience, {
+            //     "title": bufferTitle,
+            //     "company": bufferCompany,
+            //     "start_date": bufferStartDate,
+            //     "end_date": bufferEndDate,
+            //     "description": bufferDescription
+            // }]);
+            create_experience({
                 "title": bufferTitle,
                 "company": bufferCompany,
                 "start_date": bufferStartDate,
                 "end_date": bufferEndDate,
-                "description": bufferDescription
-            }]);
+                "description": bufferDescription,
+                //TODO: Add location and skills in the frontend
+                "location": "blank",
+                "skills": "blank",
+                "applicant": userId,
+            });
         }
         else{}
```

Comment:
This else condition encloses nothing! You sure you don't need to enclose the code after it?

Reply:
else{} is to prepare for next implementation for recruiters